blogposts/2021/workspaces-of-sourcegraph.md

```diff
@@ -0,0 +1,224 @@
+---
+title: The workspaces of Sourcegraph
+author: Rob Rhyne
+authorUrl: https://twitter.com/robrhyne
+publishDate: 2021-02-10T18:00+02:00
+tags: [blog, remote, design]
+slug: workspaces-of-sourcegraph
+published: true
+---
+
+The Sourcegraph design team is exploring how to improve the display of code throughout the application. As part of this exploration, we are reviewing trends in typography, color schemes, and UI complexity in the places our engineers view code the most… their editors.  While our research is far from complete, we thought you might find a breakdown of the visual characteristics of where our team writes code of interest. 
```

Comment:
why is improving the display of code throughout the application a priority for you? What problem are you addressing and do you expect the outcome of focusing on this issue to be? The reason I ask and think we can expand here is that I think the research you are doing has broad appeal. Right now, it's still pretty centered on Sourcegraph. Think of Sourcegraph as a character in your story but not the whole plot. This will make the article more interesting to more people.